### ISSUE TITLE
Date Time Picker: Time zones may need to be at a higher granularity than 1 hour

### ISSUE THREAD

**Issue by atesgoral** [CenturyLinkCloud/Cyclops#74]
There are time zones at 30 minutes and even 15 minutes bounds: https://en.wikipedia.org/wiki/List_of_UTC_time_offsets

**Comment by osbornm** [comment]
Yup I'd agree, historically we (Tier3 > CTL) have only supported hour break outs but I'm not opposed to adding the additional "missing" ones

**Comment by atesgoral** [comment]
With x4 the existing granularity, a different paradigm than a `<select>` with a tons of options might have to be investigated, too...

**Comment by osbornm** [comment]
is it really x4? or are there just a couple? a quickly looking at the list it seems like there would only be 10 maybe 12 more...

**Comment by atesgoral** [comment]
Yeah, it could as well be a heterogenous list of just all existing time zones on the planet.

**Comment by osbornm** [comment]
Is there a customer use case for this currently?
